src/common/components/Recaptcha/Recaptcha.tsx

```diff
@@ -0,0 +1,106 @@
+import angular from 'angular'
+import { useGoogleReCaptcha } from 'react-google-recaptcha-v3'
+import { react2angular } from 'react2angular'
+import { useCallback } from 'react'
+import React from 'react'
+
+const componentName = 'recaptcha'
+
+interface RecaptchaProps {
+  action: string
+  onSuccess: (componentInstance: any) => void
+  componentInstance: any
+  buttonId: string
+  buttonType?: 'submit' | 'reset' | 'button' | undefined
+  buttonClasses: string
+  buttonDisabled: boolean
+  buttonLabel: string
+  $translate: any
+  $log: any
+}
+
+export const Recaptcha = ({
+  action,
+  onSuccess,
+  componentInstance,
+  buttonId,
+  buttonType,
+  buttonClasses,
+  buttonDisabled,
+  buttonLabel,
+  $translate,
+  $log
+}: RecaptchaProps): JSX.Element => {
+  const { executeRecaptcha } = useGoogleReCaptcha();
+
+  const handleReCaptchaVerify = useCallback(async () => {
+    if (!executeRecaptcha) {
+      console.log('Execute recaptcha not yet available')
```

Comment:
I think not because the Recaptcha is meant to be an invisible (more or less) protection, not a comfort to the user. It still has the badge in the corner if it loads. It will look no different from now if it doesn't load.